cron/cron.go

```diff
@@ -56,7 +56,7 @@ func (ct *Crontab) AddJob(id string, labels map[string]string, jobType string) e
 	default:
 		logrus.Warnf("Unknown job type: %s", jobType)
 	}
-
+	logrus.Infof("Added: %s, with schedule: %s", id, schedule)
```

Comment:
I think we need to catch the error... otherwise we are logging and it could have failed.